docs/docs/manual/grelfunctions.md

```diff
@@ -413,6 +413,10 @@ As of OpenRefine 3.4.1, uniques() reorders the array items it returns; in 3.4 be
 
 Returns the current time according to your system clock, in the [ISO 8601 extended format](exploring#data-types) (converted to UTC). For example, 10:53am (and 00 seconds) on November 26th 2020 in EST returns [date 2020-11-26T15:53:00Z].
 
+###### parseDate(s dateString) {#parsedates-dateString}
+
+Returns the inputted date as a string in any valid format to a date object in the [ISO 8601 extended format](exploring#data-types) (converted to UTC). For example, "October 4th, 2021" returns [date 2021-10-04T00:00:00Z]. Generally, date formats supported by this function are stated [here](https://github.com/sisyphsu/dateparser#showcase).
+
```

Comment:
This documentation change can be removed because you are no longer introducing this `parseDate` function.